README.md

```diff
@@ -0,0 +1,390 @@
+# `pam_pkcs11`
+
+This project is a Puppet module to manage *pam_pkcs11*, a PAM module developed
+as part of the [OpenSC project] for advanced authentication with smart cards
+and other PKCS #11 cryptographic modules.
+
+[OpenSC project]: https://github.com/OpenSC/OpenSC/wiki
+
+## Table of Contents
+
+1. [Setup - The basics of getting started with pam_pkcs11](#setup)
+  * [What pam_pkcs11 affects](#what-pam_pkcs11-affects)
+  * [Setup requirements](#setup-requirements)
+  * [Beginning with pam_pkcs11](#beginning-with-pam_pkcs11)
+1. [Usage - Configuration options and additional functionality](#usage)
+1. [Reference - An under-the-hood peek at what the module is doing and how](#reference)
+1. [Limitations - OS compatibility, etc.](#limitations)
+1. [Development - Guide for contributing to the module](#development)
+
+## Setup
+
+### What `pam_pkcs11` affects
+
+In addition to installing and configuring the core `pam_pkcs11(8)` PAM module,
+this Puppet module assists in the creation of mapper files to associate users
+with their certificates.  It can also manage some aspects of the session lock
+policy through the configuration of the `pkcs11_eventmgr(8)` daemon, part of the
+*pam_pkcs11* package.
+
+### Setup Requirements
+
+#### PAM stack configuration
+
+Currently, PAM is only configured on the supported Debian-based platforms
+(Debian and Ubuntu) using `pam-auth-update`.  On other platforms, configure PAM
+to use `pam_pkcs11(8)` via other means, be that another Puppet module, a Puppet
+profile, or something else.  In most cases just the following line in the PAM
+configuration is all that is required:
+
+```
```

Comment:
Not really sure why markdownlint doesn't complain here as this should trigger rule [MD40](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md040---fenced-code-blocks-should-have-a-language-specified).

Reply:
Hmm... I looked [here](https://github.com/github/linguist/blob/master/vendor/README.md) but could not find highlighting for PAM configuration files... Any suggestions?

Reply:
I just added `text` to disable any highlighting...